magpie/job/magpie-job-hive-checkhiveup

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Would it possible to do like a simple operation against the metastore.  Like a "get list of tables" or something like that?  B/c simply checking for the pids doesn't give any assurance that the metastore is setup correctly.

For example, with Zeppelin, the test is to open a session to the zeppelin web server.  So we atleast have a tiny bit of assurance that the Zeppelin server is setup correctly and running.

Reply:
So, I haven't found a way to programmatically run a check against the metastore that wouldn't start its own Hive instance.  What I would normally do is to submit a .sql script to it however if Hive/Metastore aren't running and we submit a script, the Hive CLI will start a hive instance to run the query.  Thus it would be misleading

Reply:
I am probably not understanding something.  If you submit a sql script during a test, shouldn't the metastore already be up and running?  So it would run against that instance and test that it's working correctly?